### PR TITLE
gh-1833 fixes

### PR DESCRIPTION
**Pull request by sche**
Handles #1833

Changes proposed in this pull request:
- initiate assets transfer by selecting a balance
- fix formatting issues
- fix UI glitches

### CODE REVIEW

File: Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift [hunk 1]
```diff
@@ -186,6 +184,14 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
         }
     }
 
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        if case .balances(items: let items) = sections[indexPath.section] {
```
Comment (elgatovital) [inline Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift]: seem to conflict this pr with https://github.com/gnosis/safe-ios/pull/1844

Comment (sche) [reply Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift]: true. It was not merged in the release branch so I didn't see it

Comment (DmitryBespalov) [reply Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift]: yes, please revert 😅

**Review by DmitryBespalov — COMMENTED** [review]
needs changes so that it doesn't conflict with existing fixes in #1844

File: Multisig/UI/Assets/Transfer/TransactionViewController/TransactionViewController.swift [hunk 2]
```diff
@@ -137,7 +143,7 @@ class TransactionViewController: UIViewController {
 
     private func didEnterText(_ text: String?) {
         addressField.clear()
-        reviewButton.isEnabled = false
+        enableRiviewButtons(false)
```
Comment (DmitryBespalov) [inline Multisig/UI/Assets/Transfer/TransactionViewController/TransactionViewController.swift]: typo: `Riview` -> `Review`

File: Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift [hunk 3]
```diff
@@ -215,7 +215,7 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
         let transferFundsVC = TransactionViewController()
         transferFundsVC.tokenBalance = balance
         let ribbon = ViewControllerFactory.ribbonWith(viewController: transferFundsVC)
-        show(ribbon, sender: self)
+        present(ViewControllerFactory.modal(viewController: ribbon), animated: true)
```
Comment (DmitryBespalov) [inline Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift:218]: this adds the 'close' button to the modal, right?

**Review by DmitryBespalov — CHANGES_REQUESTED** [review]
please fix typo

File: Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift (hunk 3 above)
Comment (sche) [reply Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift:218]: yes

File: Multisig/UI/Assets/Transfer/TransactionViewController/TransactionViewController.swift (hunk 2 above)
Comment (sche) [reply Multisig/UI/Assets/Transfer/TransactionViewController/TransactionViewController.swift]: will change

**Review by DmitryBespalov — APPROVED** [review]
Looks good! 🚀

**Pull request merged**